x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -76,14 +77,6 @@
                 }
             }
         },
-        {
-            "convert": {
-                "field": "suricata.eve.http.hostname",
-                "target_field": "destination.domain",
-                "type": "string",
-                "ignore_missing": true
-            }
-        },
         {
             "convert": {
                 "field": "suricata.eve.http.http_refer",
```

Comment:
This one's straightforward as well. Any reason why you're not renaming it?

Reply:
Nope, just skipped it by mistake. Good catch